packages/smplayer-deb/smplayer-deb.pacscript

```diff
@@ -1,7 +1,8 @@
 name="smplayer-deb"
 pkgname="smplayer" # if this is a -git, -bin, etc. package, this would be the package name without extension
-version="21.10.0"
-url="https://download.opensuse.org/repositories/home:/smplayerdev/xUbuntu_21.04/amd64/smplayer_${version}-1+2.1_amd64.deb"
+repology="smplayer"
+version="22.2.0-1"
+url="https://download.opensuse.org/repositories/home:/smplayerdev/Debian_9.0/amd64/smplayer_${version}+1.1_amd64.deb"
```

Comment:
Why is this download link from opensuse? Smplayer ships assets in [GitHub](https://github.com/smplayer-dev/smplayer/releases) too

Reply:
Never knew that!